pkg/test/env/istio.go

```diff
@@ -89,6 +89,9 @@ var (
 
 	CrdsFilesDir = path.Join(ChartsDir, "istio-init/files")
 
+	// IstioAlphaInstallDir specifies the base directory for the alpha installer
+	IstioAlphaInstallDir = path.Join(GOPATH.ValueOrDefault(build.Default.GOPATH), "/src/istio.io/installer")
```

Comment:
Can you simply name this InstallerDir? Presumably we don't have a naming conflict here, and it should end up being the "right name" at the limit.